File: app/src/main/java/cn/surine/schedulex/app_widget/BoardCastSender.java
package cn.surine.schedulex.app_widget;

import android.content.Context;
import android.content.Intent;

import cn.surine.schedulex.app_widget.data.Actions;

public class BoardCastSender {
    public static void send(Context context, Intent intent) {
        context.sendBroadcast(intent);
    }

    public static void notifyWidget(Context context) {
        context.sendBroadcast(new Intent(Actions.UPDATE));
    }
}

File: app/src/main/java/cn/surine/schedulex/app_widget/core/BoardCastSender.kt
package cn.surine.schedulex.app_widget.core

import android.content.Context
import android.content.Intent
import cn.surine.schedulex.app_widget.data.Actions

object BoardCastSender {

    fun send(context: Context, intent: Intent?) {
        context.sendBroadcast(intent)
    }

    fun notifyWidget(context: Context) {
        context.sendBroadcast(Intent(Actions.UPDATE))
    }
}